lib/v2/components/flat_button_long.dart

```diff
@@ -14,6 +14,8 @@ class FlatButtonLong extends StatelessWidget {
   Widget build(BuildContext context) {
     return SizedBox(
       width: double.infinity,
+      //TODO(raul): Please fix this ASAP
```

Comment:
@RaulUrtecho  i left a bunch of TODOs like this ignoring lint that need to be fixed ASAP after we merge this pr 
@ggui004 some for you too